Packs/Anomali_ThreatStream/TestPlaybooks/playbook-ThreatStream-Test.yml

```diff
@@ -735,7 +737,8 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-    continueonerrortype: ""
+    continueonerror: true
```

Comment:
porque?

Reply:
there's an issue with this command (not the one I modified, a different one) on the 3rd party product side (happens also in their UI)
so I continue on error, and skip the condition task that validates the command in the TPB until they resolve it...
@bakatzir FYI

Reply:
hmm are we tracking it an any way? maybe we can open an internal ticket?

Reply:
I will open a ticket